roles/ara_api/tasks/config.yaml

```diff
@@ -40,17 +40,16 @@
         ara_api_secret_key: "{{ config[ara_api_env]['SECRET_KEY'] }}"
       no_log: "{{ ara_api_secure_logging }}"
 
-# If no secret key has been provided and this is the first time we are
-# running, generate a new random secret key that will be persisted in the
+# If no secret key has been provided or it is not present in the current
+# configuration, generate a new random secret key that will be persisted in the
 # configuration file.
 - when:
-    - ara_api_secret_key is none
-    - not settings_stat.stat.exists
+    - ara_api_secret_key is none or ara_api_secret_key == ''
   block:
     - name: Generate a random secret key
-      environment:
-        PATH: "{{ path_with_virtualenv }}"
-      command: "{{ ara_api_python_command }} -c 'from django.utils.crypto import get_random_string; print(get_random_string(length=50))'"
+      shell: |
+        set -o pipefail
+        tr -dc A-Za-z0-9 </dev/urandom | head -c 50 ; echo ''
```

Comment:
I remember pulling the previous implementation from a django recommendation but my understanding is that in the context of podman, django isn't installed on the host but instead in the container so we can't run the command ?